index.html

```diff
@@ -159,105 +166,108 @@ <h2>Multi-Post Stories</h2>
-                Look through some of my work and<br>
-                experience! If you like what you see and<br>
-                have a project you need coded, don’t<br>
-                hestiate to contact me.
-            </p>
         </div>
-        <div class="social-media">
-            <h3 class="contact">LET'S CONNECT </h3>
-            <ul class="social-link">
-                <li class="tweet"><a href="#">
-                        <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
-                    </a>
-                </li>
-                <li><a href="#">
-                        <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
-                    </a>
-                </li>
-                <li><a href="#">
-                        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
-                    </a>
-                </li>
-                <li><a href="#">
-                        <i class="fa fa-github fa-2x" aria-hidden="true"></i>
-                    </a>
-                <li><a href="#">
-                        <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
-                    </a>
-                </li>
-            </ul>
-        </div>
-        <button type="button">Get My Resume</button>
-        </div>
-
-        <div class="skills">
-
-            <div class="Language">
-                <h3>Languages</h3>
-                <i class="fa fa-angle-down" aria-hidden="true"></i>
+    </section>
+    <section class="duo">
+        <section class="about_section">
+            <div class="about_me_title">
+                <h2 class="abt-heading">About Myself</h2>
+
+                <p class="abt-desc">
+                    Hello I’m a software developer! I can help
+                    you build a <br> product, feature or website 
+                    Look through some of my <br> work and
+                    experience! If you like what you see and
+                    have <br> a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+                <div class="social-media-2">
+                    <h3 class="contact">LET'S CONNECT </h3>
+                    <ul class="social-link">
+                        <li class="tweet"><a href="#">
+                                <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+                            </a>
+                        </li>
+                        <li><a href="#">
+                                <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                            </a>
+                        </li>
+                        <li><a href="#">
+                                <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+                            </a>
+                        </li>
+                        <li><a href="#">
+                                <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                            </a>
```

Comment:
- [ ] Please ensure that the `about me` part takes 40% of the page and the `languages, skills and frameworks'` part takes the other 60% of the section as shown in the Figma Design :100: 

- [ ] There are also some other elements that need to be appropriately styled so your design looks similar to the Figma design. I have pointed them out in the image below :100: 

- Your Design
![image](https://user-images.githubusercontent.com/102699958/230686544-909f4457-d7b0-404b-a666-c82c08b54163.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/230458000-47622cf2-9c23-433a-b3ee-7b967202b742.png)